### PR TITLE
MULE-18487: Add operation with ee:transform to smart connector

### PR DESCRIPTION
**Pull request by szaffarano**

### CODE REVIEW

**Comment by szaffarano** [comment]
--validate

**Comment by szaffarano** [comment]
--validate

File: mule-test-smart-connectors-parent/smart-connector-using-core/pom.xml [hunk 1]
```diff
@@ -19,4 +19,13 @@
         <formatterConfigPath>../../formatter.xml</formatterConfigPath>
     </properties>
 
+    <dependencies>
+        <dependency>
+            <groupId>com.mulesoft.mule.runtime.modules</groupId>
+            <artifactId>mule-module-spring-config-ee</artifactId>
```
Comment (elrodro83) [inline mule-test-smart-connectors-parent/smart-connector-using-core/pom.xml:25]: cannot have an ee dependency in CE!

File: mule-test-smart-connectors-parent/smart-connector-using-core/src/main/resources/module-using-core.xml [hunk 2]
```diff
@@ -80,4 +82,19 @@
         <output type="string"/>
     </operation>
 
+    <operation name="set-payload-ee">
```
Comment (elrodro83) [inline mule-test-smart-connectors-parent/smart-connector-using-core/src/main/resources/module-using-core.xml:85]: this test belongs in ee

**Pull request closed without merging**